### PR TITLE
Implement EnvironmentProperties for remaining non-jdbc connectors

### PR DESCRIPTION
**Pull request by ejeffrli**
Refactor EnvironmentProperties so federation-sdk has no information on connector-specific properties

### CODE REVIEW

File: athena-cloudera-impala/src/main/java/com/amazonaws/athena/connectors/cloudera/ImpalaEnvironmentProperties.java [hunk 1]
```diff
@@ -17,15 +17,22 @@
...
-public class ImpalaEnvironmentProperties extends SaphanaEnvironmentProperties
+public class ImpalaEnvironmentProperties extends JdbcEnvironmentProperties
 {
     @Override
     protected String getConnectionStringPrefix(Map<String, String> connectionProperties)
     {
         return "impala://jdbc:impala://";
     }
+
+    protected String getDatabase(Map<String, String> connectionProperties)
```
Comment (burhan94) [inline athena-cloudera-impala/src/main/java/com/amazonaws/athena/connectors/cloudera/ImpalaEnvironmentProperties.java:35]: `@Override?`

File: athena-docdb/src/main/java/com/amazonaws/athena/connectors/docdb/DocDBEnvironmentProperties.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+import com.amazonaws.athena.connector.lambda.EnvironmentProperties;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class DocDBEnvironmentProperties extends EnvironmentProperties
+{
+    private static final String JDBC_PARAMS = "JDBC_PARAMS";
+    private static final String DEFAULT_DOCDB = "default_docdb";
+    @Override
```
Comment (burhan94) [inline athena-docdb/src/main/java/com/amazonaws/athena/connectors/docdb/DocDBEnvironmentProperties.java:35]: nit: spacing between global vars and methods

File: athena-neptune/src/main/java/com/amazonaws/athena/connectors/neptune/NeptuneEnvironmentProperties.java [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    private static final String CLUSTER_RES_ID = "CLUSTER_RESOURCE_ID";
+    private static final String NEPTUNE_GRAPHTYPE = "neptune_graphtype";
+    private static final String GRAPH_TYPE = "GRAPH_TYPE";
+
+    @Override
+    public Map<String, String> connectionPropertiesToEnvironment(Map<String, String> connectionProperties)
+    {
+        Map<String, String> environment = new HashMap<>();
+
+        environment.put(NEPTUNE_ENDPOINT, connectionProperties.get("HOST"));
```
Comment (burhan94) [inline athena-neptune/src/main/java/com/amazonaws/athena/connectors/neptune/NeptuneEnvironmentProperties.java]: we should consider putting the unified connector parameters into a single utility class

**Review by burhan94 — CHANGES_REQUESTED** [review]

File: athena-neptune/src/main/java/com/amazonaws/athena/connectors/neptune/NeptuneEnvironmentProperties.java (hunk 3 above)
Comment (ejeffrli) [reply athena-neptune/src/main/java/com/amazonaws/athena/connectors/neptune/NeptuneEnvironmentProperties.java]: sounds good. In athena-federation-sdk?

**Review by burhan94 — APPROVED** [review]

**Pull request merged**